packages/web/src/components/card/desktop/Card.tsx

```diff
@@ -146,7 +152,8 @@ const Card = ({
   trackCount,
   onClickReposts,
   onClickFavorites,
-  onClick
+  onClick,
+  link
```

Comment:
Do we want to stick to same href pattern?

Reply:
yeah, that would be better, can change!